content/en/logs/indexes/_index.md

```diff
@@ -104,3 +131,4 @@ Multiple indexes also provide the ability to define access rules on the data con
 [7]: https://docs.datadoghq.com/logs/archives/
 [8]: https://docs.datadoghq.com/logs/logs_to_metrics/
 [9]: /account_management/rbac
+[10]: /logs/processing/processors/?tab=ui#trace-remapper
```

Comment:
Might be better to link to https://docs.datadoghq.com/tracing/advanced/connect_logs_and_traces/?tab=java#pagetitle, what do you think?